### PR TITLE
Mapping for potions and trades factories

### PR DESCRIPTION
**Pull request by urielsalis**

### CODE REVIEW

File: mappings/net/minecraft/village/Trades.mapping [hunk 1]
```diff
@@ -4,26 +4,49 @@ CLASS avf net/minecraft/village/Trades
 		FIELD b price I
 		FIELD c maxUses I
 		FIELD d experience I
+		METHOD <init> (Lbbm;I)V
+			ARG 2 price
+		METHOD <init> (Lbbm;III)V
+			ARG 2 price
+			ARG 3 rmaxUses
```
Comment (liach) [inline mappings/net/minecraft/village/Trades.mapping]: watch the `r` next time

Comment (Prospector) [inline mappings/net/minecraft/village/Trades.mapping]: the age old enigmatic R

Comment (Prospector) [reply mappings/net/minecraft/village/Trades.mapping]: Shouldn't maxUses be like stock or stockSize or stockedAmount or something? Does it tell how many the villager has left to trade or how many total they can trade if the player has traded 0 so far?

Comment (liach) [reply mappings/net/minecraft/village/Trades.mapping]: No. Vanilla just calls it `maxUses`, which is the same as old maxUses in villager trades. The refill logic only impacts price in some sort of non-serialized matter.

**Review by Prospector — APPROVED** [review]

**Review by raphydaphy — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Pull request merged**